_posts/2015-09-17-Rust-1.3.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: "Announcing Rust 1.3"
+author: The Rust Core Team
+---
+
+The gear keeps turning: we're releasing Rust 1.3 stable today! As always, read
```

Comment:
Whilst I'd normally agree, I think it's a pun (Rust logo being a gear and all...)